docs/antctl.md

```diff
@@ -140,26 +135,6 @@ working directory, using the current timestamp as a suffix. The command also
 provides additional flags to filter the results: run `antctl supportbundle
 --help` for the full list.
 
-The collected support bundle will include the following (more information may be
-included over time):
-
-* cluster information: description of the different K8s resources in the cluster
-  (Nodes, Deployments, etc.).
-* Antrea Controller information: all the available logs (contents will vary
-  based on the verbosity selected when running the controller) and state stored
-  at the controller (e.g. computed NetworkPolicy objects).
-* Antrea Agent information: all the available logs from the agent and the OVS
-  daemons, network configuration of the Node (e.g. routes, iptables rules, OVS
-  flows) and state stored at the agent (e.g. computed NetworkPolicy objects
-  received from the controller).
-
-**Be aware that the generated support bundle includes a lot of information,
-  including logs, so please review the contents of the directory before sharing
-  it on Github and ensure that you do not share anything sensitive.**
-
-The `antctl supportbundle` command can also be run inside a Controller or Agent
-Pod, in which case only local information will be collected.
-
```

Comment:
ditto, I feel it should be kept since these parts are still valid information. 
This section is about Kubernetes cluster support bundle, I think it's OK to leave them as it is. Let's see what's @wenyingd 's thoughts.